docs/sources/reference/commandline/cli.md

```diff
@@ -1608,8 +1628,8 @@ deleted.
 
 #### Removing tagged images
 
-Images can be removed either by their short or long IDs, or their image
-names. If an image has more than one name, each of them needs to be
+An image can be removed by its short or long ID, its tag, or its digest. If an
```

Comment:
You can remove an image using its short or long ID, its tag, or its digest. If an image has one or more tag or digest reference, you must remove all of them before you can remove the image itself.